skimage/segmentation/_join.py

```diff
@@ -1,6 +1,7 @@
 import numpy as np
 from .._shared.utils import deprecated
-from ._remap import _map_array
+from ..util import map_array
+from ..util._map_array import ArrayMap
```

Comment:
```suggestion
from ..util._map_array import map_array, ArrayMap
```

I think it's better to use direct imports in our own code...

Reply:
oh interesting I didn't know.

Reply:
It's a new in-trial policy from #3397 =)